actioncable/app/assets/javascripts/action_cable/connection.coffee

```diff
@@ -89,19 +97,31 @@ class ActionCable.Connection
 
     open: ->
       ActionCable.log("WebSocket onopen event")
-      @disconnected = false
-      @subscriptions.reload()
+      # This is in the case that the browser client opens
+      # the connection regardless of whether a protocol was 
+      # negotiated successfully or not.
+      if not @isProtocolSupported()
+        @close()
+      else
+        ActionCable.log("WebSocket using '" + @protocol() + "' subprotocol.")
```

Comment:
There is a log in the initial open which shows the available protocols. The log above is showing the actual protocol chosen by the server.